### PR TITLE
interface avoids registered exit handlers on Release shutdown

### PR DESCRIPTION
**Pull request by AndrewMeadows**
- Release build of interface calls _exit() on shutdown

### CODE REVIEW

**Comment by ZappoMan** [comment]
Like fixes this issue - https://app.asana.com/0/26225263936266/98461007706698

**Comment by AndrewMeadows** [comment]
To test:

(1) Verify interface does not crash on quit.
(2) Verify interface saves settings on quit.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4bb705360dddc800836c6ee16f4d16baeb064947 available at: https://gist.github.com/bc437a0cd9e8b9a5a49c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7378.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7378.dmg

File: interface/src/main.cpp [hunk 1]
```diff
@@ -163,5 +163,9 @@ int main(int argc, const char* argv[]) {
     Application::shutdownPlugins();
 
     qCDebug(interfaceapp, "Normal exit.");
+#ifndef DEBUG
```
Comment (jherico) [inline interface/src/main.cpp]: I've seen deadlocks where the system seems to be hung on shutting down the server.  You might consider moving this to line 160

**Comment by ZappoMan** [comment]
ALT-F4 crash is gone... and settings are saved.

**Pull request merged**